config/releases.exs

```diff
@@ -22,6 +22,12 @@ config :logger, :console,
 # Use Jason for JSON parsing in Phoenix
 config :phoenix, :json_library, Jason
 
+secret_key_base =
+  System.get_env(
+    "SECRET_KEY_BASE",
+    "d2cgmPzW+bqVjs99FUeKJ0kOm0w8EZBvLS7UBM8EHi6uBKgW2oBAa9pR2KSu8Z2W"
+  )
+
 presto_db = [url: "http://localhost:8080", catalog: "hive", schema: "default"]
```

Comment:
do we set ourselves up for success by making this presto_db url already a `System.get_env("PRESTO_URL" || "localhost")`?

Reply:
Yeah definitely should be getting this value from `System`. Nice catch.